website/static/js/pages/profile-settings-addons-page.js

```diff
@@ -1,7 +1,15 @@
+'use strict';
+
+require('css/user-addon-settings.css');
 var $ = require('jquery');
+var ko = require('knockout');
 var bootbox = require('bootbox');
-var osfHelpers = require('osfHelpers');
+var Raven = require('raven-js');
+var $osf = require('osfHelpers');
```

Comment:
For forward-compatibility with https://github.com/CenterForOpenScience/osf.io/issues/2143, let'd do `require('js/osfHelpers')` moving forward.